docs/docs/howto/secure-image-store.md

```diff
@@ -0,0 +1,119 @@
+
+Add the following annotation to one of your pod definitions:
+
+```yaml
+metadata: # v1.Pod, v1.PodTemplateSpec
+  annotations:
+    contrast.edgeless.systems/image-store-size: 250Gi
+```
+
+Rerun `contrast generate` and reapply your deployment for the change to take effect:
+
+```bash
+contrast generate resources/
+kubectl apply -f resources/
+```
+
+### Disabling the image store for a single pod
+
+To disable the secure image store for a specific pod, set the value of the shown annotation to `0`, without a unit:
+
+```yaml
+metadata: # v1.Pod, v1.PodTemplateSpec
+  annotations:
+    contrast.edgeless.systems/image-store-size: 0
+```
+
+Rerun `contrast generate` and reapply your deployment for the change to take effect:
+
+```bash
+contrast generate resources/
+kubectl apply -f resources/
+```
+
+### Disabling the image store globally
+
+To disable the image store globally, specify the `--skip-image-store` flag in the `contrast generate` command, then reapply your deployment:
+
+```bash
+contrast generate --skip-image-store resources/
+kubectl apply -f resources/
+```
+
+### Advanced configuration
+
+If you need further customization - for example because you require the use of a custom [CSI driver](https://kubernetes-csi.github.io/docs/drivers.html) -
+you can disable the image store (either [for a single pod](#disabling-the-image-store-for-a-single-pod) or [globally](#disabling-the-image-store-globally)), and instead inject your own image store.
+
+Internally, whether or not the image store is used depends exclusively on the presence of the magic device `/dev/image_store`.
+If a device with this name is present in the pod-VM, it will be prepared, mounted and used as the secure image store.
+This requires that the device is bound to a [sidecar container](https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/), a type of init container which isn't required to exit.
+To the definition of the pod for which you wish to handle the secure image store manually, add a sidecar `initContainer` and an ephemeral volume:
+
+```yaml
+spec: # v1.Pod, v1.PodTemplateSpec
+  initContainers:
+    - command:
+        - /usr/local/bin/bash
+        - "-c"
+        - sleep infinity
+      image: "ghcr.io/edgelesssys/bash"
+      name: pvc-holder
+      resources:
+        limits:
+          memory: 50Mi
+        requests:
+          memory: 50Mi
+      restartPolicy: Always
+      securityContext:
+        privileged: true
+      volumeDevices:
+        - devicePath: /dev/image_store
+          name: image-store
+  volumes:
+    - ephemeral:
+        volumeClaimTemplate:
+          spec:
+            accessModes:
+              - ReadWriteOnce
+            resources:
+              requests:
+                storage: 10Gi
+            volumeMode: Block
+      name: image-store
+```
+
+The configuration shown above corresponds to the one generated by `contrast generate` when the image store feature is enabled.
```

Comment:
This won't be kept in sync, so we shouldn't state this. Just describe what it provides (a volume that is used to pull images).